src/conic_form.jl

```diff
@@ -0,0 +1,270 @@
+mutable struct ConeData
+    f::Int # number of linear equality constraints
+    l::Int # length of LP cone
+    q::Int # length of SOC cone
+    qa::Vector{Int} # array of second-order cone constraints
+    s::Int # length of SD cone
```

Comment:
This is specific to SCS.
Part of it is in the `num_rows::Vector{Int}` field of the `GeometricConicForm` struct.
Solvers like SCS that need metadata like `qa`, `sa`, ... should do it in `preprocess` like SCS does.